templates/geosafe/analysis/create.html

```diff
@@ -484,6 +484,38 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
     </div>
 {% endblock %}
 
+    {% if not user.is_authenticated %}
```

Comment:
CMIIW @boney-bun , does this block coming from Geonode's base html?
If that's the case, then I guess you can delete this diff, since create.html inherits from geonode_base.html (which have some overrides we wrote for geosafe) and it inherits from base.html. So, if my assumptions is correct, then your code should still work even if you delete this block.

Can you do a quick verify for me? Try to delete this block and see if it works. Thanks!